### PR TITLE
Article about transferring UK and CO.UK domains to DNSimple.

### PR DESCRIPTION
**Pull request by aeden**
I have added this article because the process is unique
and we receive quite a few support requests about it.

### CODE REVIEW

**Comment by arnamak** [comment]
Looks good to me, I'd maybe just suggest adding (or removing) the quotes from one of the "Press Transfer Domain" list items.

File: content/articles/transferring-uk-domain.markdown [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+
+## Approve the transfer
+
+To complete the transfer of a UK or CO.UK domain, you must change the domain's IPS TAG at your current domain registrar. Your registrar may allow you to do this through their web site, or you may need to contact them via email or phone.
+
+Request the change of the IPS TAG to `ENOM` (this is case-sensitive, so make sure it is in all capital letters).
+
+Once you've authorized the transfer, you may have to **wait up to 48 hours for the transfer to complete.**
+
+<callout>
```
Comment (jcaudle) [inline content/articles/transferring-uk-domain.markdown]: I didn't know about `<callout>`. :+1:

File: content/articles/transferring-uk-domain.markdown [hunk 2]
```diff
@@ -0,0 +1,74 @@
...
+We will only charge your credit card for the transfer once it has completed.
+</callout>
+
+## Transfer status
+
+You can go to your domain list at any time to check on the status of pending transfers. Click on "transfer in process" to view detailed information about the transfer request.
```
Comment (jcaudle) [inline content/articles/transferring-uk-domain.markdown]: It may be better to use `<label>` here around "transfer in process".

**Comment by weppos** [comment]
We already have an article about tranferring a UK domain: http://support.dnsimple.com/articles/domains-uk/

I suggest to expand/update it, rather than publishing a new one. Otherwise we will end up duplicating the articles.

**Comment by aeden** [comment]
The reason I created a new article is because I wanted a *specific* long-form article for the steps in transferring UK domains, whereas the article you mentioned is about all things UK (registering, transferring, etc). In other words, this was intentional.

**Comment by weppos** [comment]
Then I suggest to at least remove the transfer info in the big article and link to the specific one, so that we have some crosslinking in place.

**Comment by aeden** [comment]
Done.

**Pull request merged**